HIF.md

```diff
@@ -0,0 +1,810 @@
+|`0x05` | RCP internal error (reach out Silicon Labs support).
+
+[lqi]:  https://docs.silabs.com/rail/latest/rail-api/rail-rx-packet-details-t#lqi
+[rssi]: https://docs.silabs.com/rail/latest/rail-api/rail-rx-packet-details-t#rssi
+
+### `0x13 IND_DATA_RX`
+
+Receive IEEE 802.15.4 frames. The RCP will start emitting these indications
+once the radio is started with [`SET_RADIO`][rf-set]. See
+["Packet Filtering"][filter] to limit the amount of frames accepted.
+
+ - `uint16_t frame_len`  
+    Length of the received IEEE 802.15.4 frame.
+
+ - `uint8_t frame[]`  
+    Decrypted IEEE 802.15.4 frame, (see ["Security"][sec]).
+
+ - `uint64_t timestamp_rx_us`  
+    The timestamp (relative to the date of the RCP reset) when the frame has
+    been received on the device.
+
+ - `uint8_t lqi`  
+    Received Link Quality Indicator (LQI) as reported by [RAIL][lqi].
+
+ - `int8_t rx_power_dbm`  
+    Received power in dBm as reported by [RAIL][rssi].
+
+ - `uint8_t phy_mode_id`  
+    Wi-SUN PhyModeId used during reception.
+
+ - `uint16_t chan_num`  
+    Channel number used during reception.
+
+## Radio configuration
+
+PHY configuration and RCP level regional regulation enforcement. For
+frequency hopping parameters, see ["FHSS configuration"][fhss].
+
+[rf-list]: #0x22-cnf_radio_list
+[rf-set]:  #0x23-set_radio
+[rf-reg]:  #0x24-set_radio_regulation
+[rf-pow]:  #0x25-set_radio_tx_power
+
+### `0x20 REQ_RADIO_ENABLE`
+
+Start the radio module. Before this, no packet can be transmitted
+([`REQ_DATA_TX`][tx-req]) nor received ([`IND_DATA_RX`][rx]). Some
+configuration needs to be done before calling this command, typically a PHY
+must be selected with [`SET_RADIO`][rf-set], and a unicast schedule must be
+configured with [`SET_FHSS_UC`][uc].
+
+Body of this command is empty.
+
+### `0x21 REQ_RADIO_LIST`
+
+Request the list of radio configuration supported by the device, typically
+issued by the host during a startup sequence before selecting a PHY
+configuration. The RCP will answer with a series of [`CNF_RADIO_LIST`][rf-list]
+commands.
+
+Body of this command is empty.
+
+### `0x22 CNF_RADIO_LIST`
+
+List of radio configuration supported by the RCP, as configured during project
+generation. Entries are defined in groups of mode-switch compatible PHYs. For
+OFDM configurations, all MCS are defined in the same entry.
+
+ - `uint8_t entry_size`  
+    Size of a radio configuration entry in bytes. This is meant to support API
+    evolution.
+
+ - `bool list_end`  
+    If not set, the list will continue in another [`CNF_RADIO_LIST`][rf-list].
+
+ - `uint8_t count`  
+    Number of radio configuration entries in next field.
+
+ - `struct rf_config[]`  
+    - `uint16_t flags`
+        - `0x0001`: If set, this entry is in same group than the previous.
+        - `0x01FE`: Bitfield of supported OFDM MCS (from MCS0 to MCS11).
+    - `uint8_t rail_phy_mode_id`  
+       Wi-SUN PhyModeId, but for OFDM, only MCS0 exists.
```

Comment:
That isn't very clear. 

For OFDM, only set the PhyType. 
For OFDM, only set the Option number.